Makefile

```diff
@@ -244,4 +244,10 @@ vendor:
 		echo "Applying patch $$p"; \
 		git mailinfo /dev/null /dev/stderr 2<&1- < $$p | git apply --reject || exit 1; \
 	done
+	make vendor-etcd
```

Comment:
PTAL https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/rebase.sh#L929
We can remove `(cd "${dirpath}" && make vendor)` and just run `make vendor` in `microshift/`

Reply:
I worry that just running `make vendor` will have other side effects for the top level vendor dir. I can change that line to run `make vendor-etcd` though.

Reply:
Actually, looking closer at that script, it updates each directory one at a time and commits each separately. We can't do that if we run `make vendor` and using `make vendor-etcd` assumes that there is only 1 submodule.

I think there's some work to optimize that script, but maybe we can leave it for another PR to give us time to consider how we want it to work?

Reply:
I've removed this line for now. I'm trying to do is get to a point where we could run a job that tries to update all of the vendor directories and fails if any changes are found (because we want developers to update the vendor dir in their PR if they change something that needs to be revendored), but I can do that in another PR.